projects/common/src/navigation/navigation.service.ts

```diff
@@ -106,31 +102,20 @@ export class NavigationService {
    * Navigate within the app.
    * To be used for URLs with pattern '/partial_path' or 'partial_path'
    */
-  public navigateWithinApp(path: NavigationPath, navigationExtras?: WithinAppNavigationExtras): Observable<boolean> {
-    const params: InternalNavigationExtras = {
-      queryParams: this.buildParamsForNavigation(navigationExtras?.preserveParameters ?? []),
-      relativeTo: navigationExtras?.useDefaultRelativeTo
-        ? this.getCurrentActivatedRoute()
-        : navigationExtras?.relativeTo
-    };
-
-    return this.navigateInternal(path, params);
-  }
-
-  /**
-   * Navigate to an external URL.
-   * URL should be of the form 'https://url' or 'http://url'
-   */
-  public navigateExternal(
-    url: string,
-    navigationType: NavigationType = NavigationType.SameWindow
+  public navigateWithinApp(
```

Comment:
@aaron-steinfeld  Should we rip this off as well? We kind of need a method that would build the NavigationParams from these three inputs

Reply:
by rip off do you mean remove it? If so, I would agree - send everything through navigate. `NavigationParams` already supports the relative path. If the preserved params is still used we can add support for it, but I don't think it is from a quick search.

Reply:
Yeah. I wasn't sure about it either. I will put up a separate PR to remove this method and its usages.